src/components/Book.js

```diff
@@ -13,28 +13,38 @@ const Book = (props) => {
   };
 
   return (
-    <li>
-      <div>
-        <p>
-          <span>Book title :</span>
-          {title}
-        </p>
+    <li className="book">
+      <div className="left-block">
+        <div className="book-info">
+          <span className="category">{category}</span>
+          <span className="title">{title}</span>
+          <span className="author">{author}</span>
+        </div>
+        <div className="list-btn">
+          <button type="button">Comments </button>
+          |
+          <button
+            type="button"
+            onClick={handleRemoveClick}
+          >
+            Remove
+          </button>
+          |
+          <button type="button">Edit</button>
+        </div>
       </div>
-      <div>
-        <p>
-          <span>Author :</span>
-          {author}
-        </p>
+      <div className="reading-progress">
+        <div className="progress" />
+        <div className="progress-percentage">
+          <span className="percentage">76%</span>
+          <span className="completed">Completed</span>
+        </div>
       </div>
-      <div>
-        <p>
-          <span>Category :</span>
-          {category}
-        </p>
+      <div className="current-chapter">
+        <span className="heading">CURRENT CHAPTER</span>
+        <span className="chapter">Chapter 17</span>
+        <button type="button" className="update-btn">UPDATE PROGRESS</button>
```

Comment:
- [ ] Please kindly check all your **font types and colors** to ensure that they all look **EXACTLY** like the Zeplin design. I see some differences which I cannot individually highlight here. But kindly check all the text on your page to ensure that they conform to **font** **type** and **color**.

Reply:
@t-yanick,
Thank you for the insightful feedback. I really appreciate it.
Cheers